oresat_dxwifi/resources/temperature.py

```diff
@@ -12,20 +12,25 @@
 
 
 class TemperatureResource(Resource):
-    """Resource for getting the temperature of the NTC thermistor connected at AIN6"""
+    """Resource for getting the temperature of the NTC thermistor connected at AIN6
+
+    Attributes:
+        index: The index of the temperature data in the object dictionary. OD uses hexadecimal
+        adc: An instance of an olaf ADC that allows retrieval of voltage value from specified pin.
+    """
 
     # Olaf Constants ------------------------------------------------------------------------------
 
     TEMPERATURE_INDEX = 0x6001
 
     # Resource Related ----------------------------------------------------------------------------
 
-    def __init__(self, adc_thermistor_pin=6, is_mock_adc=False):
-        """Sets up the ADC. Determines which pin to use and whether the ADC is a real or mock ADC.
+    def __init__(self, adc_thermistor_pin: int = 6, is_mock_adc: bool = False):
+        """Sets up the ADC. Inputs set which pin to use and whether the ADC is a real or mock ADC.
 
         Args:
-            adc_thermistor_pin (int, optional): Pin where the voltage is read. Defaults to 6.
-            is_mock_adc (bool, optional): False=real world ADC, True=mock ADC. Defaults to False.
+            adc_thermistor_pin: Pin where the voltage is read.
+            is_mock_adc: False = real world ADC, True = mock ADC.
         """
         super().__init__()
         self.index = self.TEMPERATURE_INDEX
```

Comment:
Can something elsewhere in OLAF change `self.index` for a `TemperatureResource` instance via the `Resource` class interface? If not, then it's redundant since it's a constant in practice, and we could use `self.TEMPERATURE_INDEX` instead.